components/Heroes/BenchmarkBadge.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Grid, Row, Col } from 'react-flexbox-grid';
```

Comment:
I'm not sure if I'm a huge fan of it but we could definitely do it. I don't particularly like the grid system because not everything divides well into 12's but I don't see why we couldn't do it where that isn't an issue.